package.json

```diff
@@ -22,6 +22,12 @@
     "buffer-replace": "1.0.0",
     "eslint": "6.2.2",
     "eslint-config-prettier": "6.1.0",
+    "eslint-config-react-app": "5.0.1",
+    "eslint-config-synacor": "3.0.4",
+    "eslint-plugin-ember": "7.0.0",
+    "eslint-plugin-node": "9.2.0",
+    "eslint-plugin-react-app": "6.0.1",
+    "eslint-plugin-vue": "5.2.3",
```

Comment:
These dependencies are not necessary to run the project. Maybe they were required by your global eslint config?

Reply:
This is currently in because `eslint` needed it when running the pre-commit-hook. I will investigate it furhter later.

Reply:
Removed then right now, lets see if `circleci` runs through.